lib/x11utils.pm

```diff
@@ -298,6 +298,8 @@ sub select_user_gnome {
     elsif (match_has_tag('displaymanager-user-selected')) {
         if ($myuser =~ 'bernhard') {
             send_key 'ret';
+            # sometimes the system is slow, need wait several seconds for the screen to change.
+            wait_still_screen 5;
```

Comment:
I think this is performance issue. If add wait time to this common file. All of others cases will waste 5s.

Reply:
We need ensure test won't be blocked, I can limit the arch for s390x since currently only test on s390x have such issues. Later if other platform has the issue then remove the condition is ok.